backend-agent/main.py

```diff
@@ -21,6 +23,8 @@
 
 load_dotenv()
 
+app.config['SQLALCHEMY_DATABASE_URI'] = f"sqlite:///{os.getenv('DB_PATH')}"
```

Comment:
if the users don't change the `DB_PATH` in `.env` this is probably gonna raise an exception at runtime. Better to add a control before to verify if the path exists (if not, we raise an exception and ask to set an existing path for the db)